Cubical/HITs/Pushout/KrausVonRaumer.agda

```diff
@@ -0,0 +1,147 @@
+{-
+
+An induction principle for paths in a pushout, described in
+Kraus and von Raumer, "Path Spaces of Higher Inductive Types in Homotopy Type Theory"
+https://arxiv.org/abs/1901.06022
+
+-}
+
+{-# OPTIONS --cubical --safe #-}
+module Cubical.HITs.Pushout.KrausVonRaumer where
+
+open import Cubical.Foundations.Everything
+open import Cubical.Data.Sigma
+open import Cubical.HITs.Pushout.Base
+
+private
+  flippedHomotopyIsInverse : ∀ {ℓ} {A : Type ℓ} {f : A → A} (p : ∀ a → f a ≡ a)
```

Comment:
How about moving this to Foundations.Function and calling it "homotopySymInv" or something similar?

Reply:
Done! Sadly the other path lemmas are somewhat weird... I hoped that there was a way to build them out of lemmas already in the library but I didn't come up with anything.